src/DurableTask.AzureStorage/Tracking/AzureTableTrackingStore.cs

```diff
@@ -1190,8 +1200,21 @@ Type GetTypeForTableEntity(DynamicTableEntity tableEntity)
             if (historyEntity.Properties.TryGetValue(blobPropertyName, out EntityProperty blobProperty))
             {
                 // This is a large message
-                string blobName = blobProperty.StringValue;
-                string blobUrl = this.messageManager.GetBlobUrl(blobName);
+                string blobData = blobProperty.StringValue;
+
+                // We now store blobs as absolute URIs to minimize chance of 'blob not found' errors
```

Comment:
This comment is too vague. I think it needs to be more specific to help future maintainers understand the implications:

```suggestion
                // As of 1.15.1, we store blobs as absolute URIs so that we don't have to assume what task hub they were created for.
                // This is necessary in cases where a common tracking store is shared across multiple task hubs.
```